files/es/games/techniques/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Técnicas para el desarrollo de juegos web
+slug: Games/Techniques
+translation_of: Games/Techniques
+---
+
+{{GamesSidebar}}
+
+Esta página muestra las técnicas básicas esenciales para cualquiera que quiera desarrollar juego web usando tecnologías web abiertas.
+
+- [Utilizar scripts asíncronos para asm.js (en-US)](/en-US/docs/Games/Techniques/Async_scripts)
```

Comment:
The render bot makes the redirection automatically if doesn't exists and nothing if exists

```suggestion
- [Utilizar scripts asíncronos para asm.js](/es/docs/Games/Techniques/Async_scripts)
```